sites/default/sqlconf.php

```diff
@@ -1,26 +1,29 @@
 <?php
 //  LibreEHR
-//  MySQL default Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
+//  MySQL Config
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
 $pass	= 'libreehr';
 $dbase	= 'libreehr';
 
+global $disable_utf8_flag;
+$disable_utf8_flag = false;
+
 $sqlconf = array();
 global $sqlconf;
 $sqlconf["host"]= $host;
```

Comment:
@Gicheha please add this file to your gitignore file. this is not suppose to be part of this commit.I think we corrected this last time